site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst

```diff
@@ -0,0 +1,43 @@
+Doctrine MongoDB ODM release 1.0.5
+==================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.0.5 <https://github.com/doctrine/mongodb-odm/releases/tag/1.0.5>`__.
+
+Bug fixes in this release
+-------------------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/master/CHANGELOG-1.0.md#105-2016-02-16>`__.
+A full list of issues and pull requests included in this release may be found in the
+`1.0.5 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.0.5>`__.
+
+Installation
+------------
+
+You can install the latest version using the following ``composer.json`` definitions:
+
+.. code-block:: json
+
+  {
+      "require": {
+          "doctrine/mongodb-odm": "^1.0.5"
+      }
+  }
+
+Doctrine MongoDB ODM and PHP 7
+------------------------------
+
+While ODM still relies on old Mongo driver and no dates are scheduled for the 2.0 release,
```

Comment:
I would rephrase as "legacy MongoDB driver (i.e. `ext-mongo`)" and link the extension name to https://pecl.php.net/package/mongo